malloc.c

```diff
@@ -1679,7 +1679,7 @@ morecore(int bucket)
 	    /* It's our first time.  Initialize ourselves */
 	    were_called = 1;	/* Avoid a loop */
 	    if (!MallocCfg[MallocCfg_skip_cfg_env]) {
-		char *s = getenv("PERL_MALLOC_OPT"), *t = s, *off;
+		char *s = PerlEnv_getenv("PERL_MALLOC_OPT"), *t = s, *off;
```

Comment:
This is malloc initialization, right? That implies there must be guarantees, at least by way of comments, that PerlEnv_getenv itself and all work up to the point PL_Env is setup avoid any calls to Perl_malloc. If such protection already exists, it would be valuable to call it out in the commit message.